src/schema/vulnerability.schema.json

```diff
@@ -9,7 +9,7 @@
       "recommendation": { "type": "string" },
       "cve": {
         "type": "string",
-        "pattern": "/^CVE-[0-9]{4}-[0-9]{5}$/i"
+        "pattern": "/^CVE-[0-9]{4}-[0-9]{4,7}$/i"
```

Comment:
Oh wait, I'm still seeing the original commit, looks unexpected.

Reply:
Woopsie, forgot `git add` :upside_down_face:

Reply:
Ah thanks for fixing! 🙏